includes/basics/baseurl.md

```diff
@@ -39,8 +39,13 @@ Within this address, you'll find APIs (collections of related endpoints) coverin
         <td><code class="language-http">{{ stache.config.gift_api_suffix }}</code></td>
         <td>Used to manage gift information, including related entities such as acknowledgements, fundraisers, and receipts.</td>
       </tr>
+      <tr>
+        <td class="nowrap">Peer-to-Peer (Beta)</td>
+        <td><code class="language-http">{{ stache.config.peer_to_peer_api_suffix }}</code></td>
+        <td>Used to create supporter pages, and query campaigns, donors, donations, and teams.</td>
```

Comment:
I copied similar wording from the other entries in the table. I agree with your suggestions but wouldn't we need to also change the other entries to match?

Examples:

Constituent API
Used to manage constituent information, including related entities such as addresses, phones, emails, and notes.

Gift API
Used to manage gift information, including related entities such as acknowledgements, fundraisers, and receipts.